bundles/framework/publisher2/view/PublisherSidebar.js

```diff
@@ -466,6 +487,9 @@ Oskari.clazz.define('Oskari.mapframework.bundle.publisher2.view.PublisherSidebar
             }
         },
 
+        isEnabled: function () {
+            return this._enabled;
+        },
```

Comment:
We probably don't need this._enabled or anything using it anymore?

Reply:
We don't. Just scrap left over from two or three iterations ago 😛